app/actors/hyrax/actors/apply_order_actor.rb

```diff
@@ -27,15 +27,15 @@ def sync_members(env, ordered_member_ids)
         # @see Hyrax::Actors::AddToWorkActor for duplication
         def cleanup_ids_to_remove_from_curation_concern(curation_concern, ordered_member_ids)
           (curation_concern.ordered_member_ids - ordered_member_ids).each do |old_id|
-            work = Valkyrie.config.metadata_adapter.query_service.find_by_alternate_identifier(alternate_identifier: old_id, use_valkyrie: false)
+            work = Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: old_id, use_valkyrie: false)
             curation_concern.ordered_members.delete(work)
             curation_concern.members.delete(work)
           end
         end
 
         def add_new_work_ids_not_already_in_curation_concern(env, ordered_member_ids)
           (ordered_member_ids - env.curation_concern.ordered_member_ids).each do |work_id|
-            work = Valkyrie.config.metadata_adapter.query_service.find_by_alternate_identifier(alternate_identifier: work_id, use_valkyrie: false)
+            work = Hyrax.metadata_adapter.query_service.find_by_alternate_identifier(alternate_identifier: work_id, use_valkyrie: false)
```

Comment:
Why does this one keep metadata_adapter?

Reply:
oh, no reason. i'll fix it